src/main/java/no/nav/fo/veilarbregistrering/registrering/resources/RegistreringResource.java

```diff
@@ -124,6 +127,30 @@ public BrukerRegistreringWrapper hentRegistrering() {
         return brukerRegistreringWrapper;
     }
 
+    @GET
+    @Path("/dummyregistrering")
+    @ApiOperation(value = "Henter dummy registrering.")
+    public OrdinaerBrukerRegistrering hentDummyRegistrering() {
```

Comment:
Kunne vi her ha laget en DTO, så vi ikke har den sterke koblingen til domenemodellen? De kan i prinsipper være like pr i dag, men om vi får gjort dette hele veien, så kan på sikt domenemodellen også bli rikere. Nå kan vi ikke gjøre noe med den, fordi den lekker helt ut

Reply:
Trenger ikke nødvendigvis gjøre det for testingen her nå, når du i prinsippet lager en stub, men når vi kobler på reelle data så hadde det vært fint.

Reply:
Har du tenkt noe rundt input? Er det FNR eller id-en til registreringen? Det er her jeg kunne tenkt meg å bruke en UUID i stedet for databaseId (ref. tidligere prat rundt design av tilstand-tabellen). Og får vi hentet ut identen til saksbehandler så det også kan lagres sammen med oppdateringen?

Reply:
Helt enig i de to første kommentarene.

For å spørre etter den aktuelle registreringen må vi vel bruke fnr? Jeg klarer ikke å se at vi klarer å finne den uten å bruke en id som er koblet til bruker.

Identen til saksbehandler kan vi få fra dekoratøren :)